generators/client/templates/angular/package.json.ejs

```diff
@@ -144,6 +144,9 @@
     <%_ if (protractorTests) { _%>
     "ts-node": "8.10.2",
     <%_ } _%>
+    <%_ if (cypressTests) { _%>
+        "cypress": "4.11.0",
```

Comment:
Is it the latest version? I ask because I think I saw yesterday a newer version when playing with the PR